documentation/themes/beastie/i18n/el.toml

```diff
@@ -77,34 +77,34 @@ other = "Download"
 other = "Download FreeBSD"
 
 [handbook]
-other = "FreeBSD Handbook"
+other = "Εγχειρίδιο FreeBSD"
 
 [handbook-desc]
-other = "A constantly evolving, comprehensive resource for FreeBSD users"
+other = "Μια συνεχώς εξελισσόμενη, περιεκτική πηγή για χρήστες του FreeBSD"
 
 [porters-handbook]
 other = "FreeBSD Porter's Handbook"
 
 [porters-handbook-desc]
-other = "A complete reference about the FreeBSD ports system"
+other = "Ένας ολοκληρωμένος οδηγός για την συλλογή Ports του FreeBSD"
 
 [faq]
-other = "FreeBSD FAQ"
+other = "Συχνές ερωτήσεις (FAQ) για το FreeBSD"
 
 [faq-desc]
-other = "FreeBSD frequently Asked Questions (FAQ) for 13.X and 12.X"
```

Comment:
[#22 - FreeBSD Documentation Portal: update the description of FreeBSD FAQ - freebsd-doc - Codeberg.org](https://codeberg.org/grahamperrin/freebsd-doc/issues/22)